### ISSUE TITLE
'Meet the Team' section

### ISSUE THREAD

**Issue by AsmahanM** [Asmahan97/User-Stories#5]
Potential clients will want to get to know who is behind the agency. They will likely  be looking for basic information such as each team member's
- Name
- Role
- Background.

The page should consider accessibility and a screen reader should be able to follow along. Consider contrasting colours and the size of the text for accessibility purposes.
